inputFiles/hydraulicFracturing/heterogeneousInSitu_base.xml

```diff
@@ -124,7 +124,7 @@
     <SurfaceElementRegion
       name="Fracture"
       defaultAperture="1.0e-4"
-      materialList="{ water, rock, fractureFilling, fracturePorosity, fracturePerm, nullSolid }"/>
+      materialList="{ water, rock, fractureFilling, fractureContact }"/>
```

Comment:
@CusiniM Is this the complete list of materials with your recent changes?

Reply:
Yes, this should be enough.